pages/en/lb3/readmes/loopback-connector-ibmi.md

```diff
@@ -0,0 +1,136 @@
+```
+
+## Configuration
+
+In LoopBack, datasources are used to store the information about your database so it can be used by the program. Use the [data source generator](https://loopback.io/doc/en/lb3/Data-source-generator.html) to add  to your application:
+
+```
+lb datasource
+```
+
+The datasource generator will then walk you through the process of setting up a datasource:
+1. `Enter the datasource name:` Any name will do, such as the name of the schema you will use or the name of your system.
+2. `Select connector for <name>:` LoopBack 3 does not have knowledge of `loopback-connector-ibmi`, so just press up on the arrow key once and select `other`.
+3. `Enter the connector's module name:` Enter `loopback-connector-ibmi`.
+4. `Install loopback-connector-ibmi`: If you haven't installed it, enter `Y`. If you have already installed it, select `n`.
+
+This will generate an entry in your `server/datasources.json` file. It should know have an entry similar to:
+
+```json
+  "test": {
+    "name": "test",
+    "connector": "loopback-connector-ibmi"
+  }
+  ```
+You should edit this entry to add information on how to connect to Db2. For `loopback-connector-ibmi`, you need to pass either a `connectionString`, or pass your `username`, `password`, and `dsn` (which will be the DSN name you set up for your ODBC driver).
+
+```json
+"test": {
+  "name": "test",
+  "connector": "loopback-connector-ibmi",
+  "connectionString": "DSN=MYDSN"
+}
+```
+
+or
+
+```json
+"test": {
+  "name": "test",
+  "connector": "loopback-connector-ibmi",
+  "dsn": "MYDSN",
+  "username": "FIRSTLAST",
+  "password": "password123"
+}
+```
+
+The following table describes the connector properties.
+
+Property| Type | Description
+---| --------| --------
+connectionString | String  | ODBC connection string for connecting to the database
+dsn            | String  | ODBC DSN to use for the connection
+username       | String  | Username on the IBM i
+password       | String  | Password on the IBM i
+schema         | String  | Specifies the default schema name that is used to qualify unqualified database objects in dynamically prepared SQL statements. The schema name is case-sensitive.
+
+**More connector properties will be added as requested by the community**
+
+Alternatively, you can create and configure the data source in JavaScript code.
+For example:
+
+```JavaScript
+var DataSource = require('loopback-datasource-juggler').DataSource;
+var DB2 = require('loopback-connector-ibmi');
+
+var config = {
+  dsn:      process.env.DSN
+  username: process.env.DB2_USERNAME,
+  password: process.env.DB2_PASSWORD,
+};
+
+var db = new DataSource(DB2, config);
+
+var User = db.define('User', {
+  name: { type: String },
+  email: { type: String },
+});
+
+// Will make sure that 'User' table has the same format as the model
+db.autoupdate('User', function(err) {
+  if (err) {
+    console.log(err);
+    return;
+  }
+
+  User.create({
+    name: 'Tony',
+    email: 'tony@t.com',
```

Comment:
Please use `example.com` domain for test email addresses, see https://loopback.io/doc/en/contrib/style-guide-es6.html#email-examples

Reply:
Thanks! Corrected in newest commit